4.6.2/Dockerfile

```diff
@@ -9,6 +9,25 @@ RUN Add-WindowsFeature Web-Server; \
 
 ADD ServiceMonitor.exe /
 
+RUN powershell -Command \
+	#download and unzip Roslyn nupkg
+	wget  https://api.nuget.org/packages/microsoft.net.compilers.2.3.1.nupkg -OutFile c:\microsoft.net.compilers.2.3.1.zip ; \	
```

Comment:
I'm not following - where did it suggest using cmdlet aliases?  I know some people like using aliases but others don't.  Some aliases are understandable IMO but some aren't.  Why wasn't `start` used over `Start-Process`?  IMO standardizing of the cmdlet name is just easier.

Reply:
It's under "Optimize Build Speed": https://docs.microsoft.com/en-us/virtualization/windowscontainers/manage-docker/optimize-windows-dockerfile#optimize-build-speed

But, regardless, I agree with you, @MichaelSimons: We should use cmdlet names instead of aliases.